calculator-service/src/main/java/example/calculator/I32.java

```diff
@@ -0,0 +1,121 @@
+/**
```

Comment:
This is a generated file, it would be probably better not to put it to the repository. If it is easy, consider to put generated sources into the `target` directory.